src/functions-and-arrays.js

```diff
@@ -1,19 +1,99 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(num1, num2) {
+  return Math.max(num1, num2);
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(arrayOfWords) {
+
+  //tests -probar los parámetros
+  if (arrayOfWords.length === 0) {
+    return null;
+  }
+
```

Comment:
Excelente trabajo @CSS1982 !!

Una consulta, no crees que con chequear el length de palabraGanadora es suficiente?

Reply:
Jasmine te pedía devolver también la palabra sino salía un error.

Reply:
Claro, vos guardas la palabra en la variable pero chequeas el length de dicha palabra y despues devolves la palabra

Reply:
Me refiero a esto:

`
  let palabraGanadora = arrayOfWords[0]

  for (let i = 1; i < arrayOfWords.length; i++) {
    const currentWord = arrayOfWords[i]
    if (currentWord.length > palabraGanadora.length) {
      palabraGanadora = currentWord
    }
  }
  return palabraGanadora;
`